doc(kic) update aws-lambda service note (#4730)
Kong/docs.konghq.com: app/_hub/kong-inc/aws-lambda/_index.md

#### Use a fake upstream service

When using the AWS Lambda plugin, the response will be returned by the plugin
itself without proxying the request to any upstream service. The service
configured for a route is ignored when using this plugin.

Versions of {{site.ce_product_name}} prior to 2.x require a service on all
routes. Even though the service will not be used, you must configure a
placeholder service for routes using this plugin. Versions after 2.x allow you
to omit service configuration.

> When using {{site.kic_product_name}}, Kubernetes Ingresses require a Service.
> Even on {{site.ce_product_name}} versions that support empty services, you will
> still need to configure a placeholder Service for the Ingress. An [ExternalName
> Service](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
> for an unresolvable domain (for example, `fake.example`) will satisfy this
> requirement without requiring a Deployment associated with the Service.

Reviewer: ```suggestion
When using {{site.kic_product_name}}, Kubernetes Ingresses require a service.
Even on {{site.ce_product_name}} versions that support empty services, you must still configure a placeholder service for the Ingress. An [ExternalName
service](https://kubernetes.io/docs/concepts/services-networking/service/#externalname)
for an unresolvable domain (for example, `fake.example`) satisfies this
requirement without requiring a Deployment associated with the service.
```
